age_compare.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p id="output"></p>
+
+<script>
+class User {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+
+  compare(user){
```

Comment:
compare is not helping us to determine what this function is doing, please rename.

Reply:
Okay I will rename it